src/Test.elm

```diff
@@ -99,26 +105,26 @@ describe untrimmedDesc tests =
     in
         if desc == "" then
             Internal.failNow
-                { description = "You must pass 'describe' a nonempty string!"
+                { description = "This `describe` has an empty description string. Let's give it a nonempty one!"
                 , reason = Test.Expectation.Invalid Test.Expectation.BadDescription
                 }
         else if List.isEmpty tests then
             Internal.failNow
-                { description = "You tried to describe '" ++ desc ++ "' but included no tests!"
+                { description = "This `describe " ++ toString desc ++ "` has no tests in it. Let's give it some!"
                 , reason = Test.Expectation.Invalid Test.Expectation.EmptyList
                 }
         else
             case Internal.duplicatedName tests of
                 Err duped ->
                     Internal.failNow
-                        { description = "The tests '" ++ desc ++ "' contains multiple tests named '" ++ duped ++ "'. Do some renaming so that tests have unique names."
+                        { description = "The tests '" ++ desc ++ "' contains multiple tests named '" ++ duped ++ "'. Let's do some renaming so that tests have unique names."
```

Comment:
How about `Let's rename them so we know which one is which.`? I don't know how formal we want to be, but it seems like we're going down the informal route.